### PR TITLE
major update website

### PR DESCRIPTION
**Pull request by francescofuggitti**
- change UI
- clean repository
- delete news
- change publication in .bib file

### CODE REVIEW

File: _pages/people.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
-
- **We are  looking for new PhD students, Postdocs, and Master students to join the team** [(see openings)]({{ site.url }}{{ site.baseurl }}/vacancies) **!**
-
-
-<!-- Jump to [staff](#staff), [master and bachelor students](#master-and-bachelor-students), [alumni](#alumni), [administrative support](#administrative-support), [lab visitors](#lab-visitors). -->
+<div class="alert alert-danger">
+We are currently looking for new PhD students, Postdocs, and Master students to join the team <a href="{{ site.url }}{{ site.baseurl }}/vacancies" class="alert-link">see openings</a>.
+</div>
 
+# Faculty.
```
Comment (marcofavorito) [inline _pages/people.md:13]: I'd remove the dot. Thoughts?

File: _pages/people.md [hunk 2]
```diff
@@ -77,9 +49,9 @@ permalink: /team/
 
 
 
-<!--## Master Students
+# Postdocs.
```
Comment (marcofavorito) [inline _pages/people.md:52]: same here

File: _pages/people.md [hunk 3]
```diff
@@ -130,10 +90,10 @@ permalink: /team/
 </div>
 {% endif %}
 
-## Alumni
+# PhD Students.
```
Comment (marcofavorito) [inline _pages/people.md:93]: same here

File: _pages/home.md [hunk 4]
```diff
@@ -5,30 +5,26 @@ excerpt: "WhiteMech at Sapienza University."
...
 **Positions available as Senior Postdoc, Junior Postdocs and PhD students.** [(more info)]({{ site.url }}{{ site.baseurl }}/vacancies) **!**
+-->
 
-
-We are grateful for funding from European Union, [ERC Advanced Grant](https://erc.europa.eu/funding/advanced-grants).
-
+We are grateful for funding from the European Union with an [ERC Advanced Grant](https://erc.europa.eu/funding/advanced-grants).
 <figure class="fourth">
-  <img src="{{ site.url }}{{ site.baseurl }}/images/logopic/Logo_WhiteMech.svg" style="width: 110px">
+  <img src="{{ site.url }}{{ site.baseurl }}/images/logopic/logo-erc.svg" style="width: 25%">
```
Comment (marcofavorito) [inline _pages/home.md:28]: I can't see this picture, maybe it needs to be renamed?

Comment (francescofuggitti) [reply _pages/home.md:28]: no, it has to be added and committed

**Comment by marcofavorito** [comment]
Another issue: I can't see the publications, it is always loading. Not sure if it is due to my network, the fact that I run the website locally, or something else.

File: _pages/people.md (hunk 1 above)
Comment (francescofuggitti) [reply _pages/people.md:13]: good point. it's the same for me.

**Comment by marcofavorito** [comment]
I like the 
image though

**Comment by francescofuggitti** [comment]
> Another issue: I can't see the publications, it is always loading. Not sure if it is due to my network, the fact that I run the website locally, or something else.

No, there can be a problem directly in the .bib file

**Comment by francescofuggitti** [comment]
> > Another issue: I can't see the publications, it is always loading. Not sure if it is due to my network, the fact that I run the website locally, or something else.
> 
> It could be a problem directly in the .bib file

The problem was due to a libraries conflict with bibbase.org

**Comment by marcofavorito** [comment]
I can't see the Sapienza logo picture:

image

**Comment by marcofavorito** [comment]
the same for the pictures of some team members
image

**Comment by marcofavorito** [comment]
image

**Comment by marcofavorito** [comment]
image

**Comment by francescofuggitti** [comment]
thanks, I added pics

**Comment by marcofavorito** [comment]
Some author lists of the publications look wrong:
image

**Comment by marcofavorito** [comment]
This PR fixes #20

**Comment by marcofavorito** [comment]
Can we also close #6 ?

**Comment by francescofuggitti** [comment]
> Some author lists of the publications look wrong:
> image
> 
> here Fabio Patrizi is missing. Similar issues in other items

Well spotted! thanks. The bib file was missing a comma 😄

**Comment by francescofuggitti** [comment]
> Can we also close #6 ?

Yes, I think so

**Pull request merged**